Gemfile.lock

```diff
@@ -45,13 +45,436 @@ GEM
     ast (2.3.0)
     autoprefixer-rails (7.1.4)
       execjs
-    aws-sdk (2.10.29)
-      aws-sdk-resources (= 2.10.29)
-    aws-sdk-core (2.10.29)
+    aws-partitions (1.21.0)
```

Comment:
Hm, I'm also not sure if this change should be part of this PR 🤔 I think this is already changed back on master. Can u try to either rebase or merge with master again?